leakcanary-android/src/main/AndroidManifest.xml

```diff
@@ -49,6 +51,7 @@
     <activity
         android:theme="@style/leak_canary_Theme.Transparent"
         android:name=".internal.RequestStoragePermissionActivity"
+        android:process=":leakcanary"
```

Comment:
Profiled leakcanary-sample in Android Studio 2.4 preview 7 and launched using RequestStoragePermissionActivity using `adb shell am`.  Turns out that while it eats some heap space in the com.example.leakcanary:leakcanary process, it's not as much as I thought.

![image](https://user-images.githubusercontent.com/1868149/27499586-650e267a-5819-11e7-87fb-2961c0611fe3.png)

:shipit: